Structure_Engine/Create/Loads/LoadCombination.cs

```diff
@@ -33,13 +37,13 @@ public static partial class Create
         /**** Public Methods                            ****/
         /***************************************************/
 
-        public static LoadCombination LoadCombination(string name, int number, IEnumerable<Tuple<double, ICase>> factoredCases)
-        {
-            return new LoadCombination { LoadCases = factoredCases.ToList(), Name = name, Number = number };
-        }
-
-        /***************************************************/
-
+        [Description("Creates a LoadCombination from two matching lists of load cases and factors.")]
```

Comment:
Creates a  load combination from two equal length lists of load cases and factors.